frontend/src/metabase/visualizations/visualizations/PivotTable.jsx

```diff
@@ -354,6 +367,7 @@ export default class PivotTable extends Component {
                       </div>
                     );
                   }}
+                  onRowsRendered={() => {}}
```

Comment:
What's happening here?

Reply:
Whoops! This was a failed attempt at following some Github thread advice to fix `Grid`/`List` not rerendering.